megamek/src/megamek/common/weapons/AreaEffectHelper.java

```diff
@@ -289,7 +289,7 @@ public static void checkInfantryDestruction(Entity entity, int distFromCenter, E
         
         int roll = Compute.d6(2);
         int result = roll + distFromCenter;
-        boolean destroyed = result > rollTarget;
+        boolean destroyed = result < rollTarget;
```

Comment:
```suggestion
        boolean destroyed = result <= rollTarget;
```

Reply:
>If the modified roll is **9 or less** for a conventional infantry unit, or is less than 7 for a battle armor unit, ..

What a confusing description. Thanks for point it out.